examples/pytorch/GATNE-T/requirements.txt

```diff
@@ -1,7 +1,7 @@
 tqdm
 numpy
-sklearn
```

Comment:
Will this change affect installing dependencies with `requirements.txt`? Have you tried it?

Reply:
Yes. In fact, current installing dependencies would fail because `sklearn` is no longer available for pip install.

Reply:
Cool